testing/execution.ts

```diff
@@ -432,7 +432,7 @@ export async function executeFormulaOrSyncWithVM<T extends PackFormulaResult | G
   formulaName: string;
   params: ParamValues<ParamDefs>;
   bundlePath: string;
-  executionContext?: ExecutionContext;
+  executionContext?: SyncExecutionContext;
```

Comment:
Hrm looks like we already had this in executeFormulaOrSyncWithRawParamsInVM but this utility isn't specific to syncs but uses `SyncExecutionContext` which is wrong. I guess isn't not causing problems at the moment but we later may need to go clean things up to parameterize this function with the type of execution context for greater accuracy.